### PR TITLE
Adds a Summon Robes Spell

### PR DESCRIPTION
**Pull request by Arthurdentist**
No Clothes is boring, why not be a sneaky git until you suddenly summon those robes and go on a rampage?

Summon Robes:
- Summons a new (and random) set of wizard gear on you
- Whatever you were wearing in the slots before is dropped on the floor
- Unsurprisingly, this spell doesn't require robes to cast. 
- For Houdini Fans (aka me), this spell will allow you to escape from a straight jacket (in a roundabout fashion)
- My code as always is terrible and I fully expect to be shouted at for this terrible excuse of a PR.

### CODE REVIEW

**Comment by ghost** [comment]
:+1: new spells and variety are always good

File: code/modules/spells/targeted/equip/robesummon.dm [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	cooldown_min = 50
+
+	compatible_mobs = list(/mob/living/carbon/human)
+
+	hud_state = "wiz_robesummon"
+
+/spell/targeted/equip_item/robesummon/New()
+	..()
+
+	switch(pick("blue","red","marisa")) //Possible Robe is initially picked here so that the spell works the first time it is cast
```
Comment (RemieRichards) [inline code/modules/spells/targeted/equip/robesummon.dm]: you can just have the `..()` of `cast()` happen AFTER the switch, and then you don't need this massive copypasta in `New()`

**Comment by Wizardcrying** [comment]
> invocation = "I PUT ON MY ROBE AND WIZARD HAT!"

I'm fucking dying

**Comment by Probe1** [comment]
How many times can it be used?

**Comment by Wizardcrying** [comment]
Probably no limit, and for good reason; you can pull off a surprise attack or (like Arthur sees this being used) to replace your current clothing, and not just once.
Please don't require this be limited.
Please.

**Comment by Probe1** [comment]
That would mean there is no way to stop a wizard without killing him.  Stripping him and putting him his day glo orange jumpsuit would result in the wizard jaunting away the second anyone walked away.

Wizard is gay enough desu.  

Feedback: I would much rather see this as the ultimate form of the spell and have either 1 or 2 additional levels, allowing it to be used once, or twice then exhausted.  Alternatively, grant the player a scroll that is consumed.

**Comment by Arthurdentist** [comment]
To be fair the vast majority of spells are actually no robes required and at least one of them (Knock) will also get you out of perma. If security locks a wizard up then they are already making a big mistake.

This spell just lets wizards be sneaky until they go nuclear. With no clothes you can be very sneaky but you will only get three 'real' spells. With this you can go loud but once you do the crew will likely be alerted, so be prepared to get valided.

**Comment by Probe1** [comment]
I still disagree despite the reasoning and it's compounding on the already unfun nature of wizards being absolutely ridiculous to catch.  It is a get out of being caught free card and people don't like jaunt, or blink, because of that reason.

**Comment by Wizardcrying** [comment]
Yeah, so fucking laser/shotgun/whatever them.
If you try to catch them you deserve whatever's coming to you.

> and it's compounding on the already unfun nature of wizards being absolutely ridiculous to catch

Too bad, Level 2 Druid.

**Comment by BestName** [comment]
@Probe1 Straitjacket + muzzle combo can still stop them without killing them can't it? And if that fails you can always get an anesthetic canister from medbay and keep them knocked out forever.

**Comment by NigglyWiggly** [comment]
> invocation = "I PUT ON MY ROBE AND WIZARD HAT!"

Arthur have I ever told you that I love you?

**Comment by Probe1** [comment]
@NigglyWiggly Arthur casts  Lvl 8 Penis of the Infinite

**Pull request merged**